ui/app/templates/components/oidc/assignment-form.hbs

```diff
@@ -0,0 +1,82 @@
+<div ...attributes>
```

Comment:
I don't believe this needs to be wrapped in a div with `...attributes` here - this whole page is rendered when we go to the `/create` or `/edit` route so there's nothing from the parent to pass down